### PR TITLE
Move Fido tutorial up a doc level

### PR DESCRIPTION
**Pull request by dstansby**
I think it makes more sense to have this at the top level, with links to the data-provider specifcic docs at the top.

### CODE REVIEW

File: docs/intro/acquiring_data/index.rst [hunk 1]
```diff
@@ -4,14 +4,534 @@
...
 
-    fido
     jsoc
     hek
     helioviewer
     sample-data
     database
+
+
+This guide outlines how to search for and download data using the `~sunpy.net.Fido` interface for search and download.
```
Comment (hayesla) [inline docs/intro/acquiring_data/index.rst:21]:
```suggestion
This tutorial outlines how to search for and download data using the `~sunpy.net.Fido` interface for search and download.
```

maybe guide is the correct phrasing here, but maybe to be consistent tutorial would be better?

File: docs/intro/acquiring_data/index.rst [hunk 2]
```diff
@@ -4,14 +4,534 @@
...
+`~sunpy.net.Fido` is a unified interface for searching and fetching solar physics data irrespective of the underlying client or webservice through which the data is obtained.
+It therefore supplies a single, easy, and consistent way to obtain most forms of solar physics data.
+
+The `~sunpy.net.Fido` object is in `sunpy.net`.
+All the examples in this guide use ``Fido``, so lets start by importing it::
```
Comment (hayesla) [inline docs/intro/acquiring_data/index.rst:26]:
```suggestion
All the examples in this tutorial use ``Fido``, so lets start by importing it::
```

File: docs/intro/acquiring_data/index.rst [hunk 3]
```diff
@@ -4,14 +4,534 @@
...
+
+    >>> from sunpy.net import Fido, attrs as a
```
Comment (hayesla) [inline docs/intro/acquiring_data/index.rst:28]: should we also mention here what "attrs" is given that its imported here too?

File: docs/intro/acquiring_data/index.rst [hunk 4]
```diff
@@ -4,14 +4,534 @@
...
+These attributes also support tab-completion to auto to auto-fill the attribute name, for example typing ``a.jsoc.aia_f<TAB>`` in a jupyter notebook will show you the available attributes that start with "aia_f".
+
+To search for data use the ``Fido.search`` method::
+
+    >>> result = Fido.search(a.Time('2012/3/4', '2012/3/6'), a.Instrument.lyra, a.Level.two) # doctest: +REMOTE_DATA
+
+this returns an `~sunpy.net.fido_factory.UnifiedResponse` object containing all the search results that match the search attributes.
+This does not download the files; we'll learn how to do that later in :ref:`downloading_data`.
+
+To see a summary of the results print the result variable that came back from the previous search::
```
Comment (hayesla) [inline docs/intro/acquiring_data/index.rst:116]:
```suggestion
To see a summary of the results, we can print the `result` variable that came back from the previous search::
```

**Review by hayesla — APPROVED** [review]

File: docs/intro/acquiring_data/index.rst [hunk 5]
```diff
@@ -4,14 +4,534 @@
...
+There are some data-provider specific tutorials that you might want to visit after going through this tutorial:
 
 .. toctree::
     :maxdepth: 1
 
-    fido
     jsoc
     hek
     helioviewer
     sample-data
```
Comment (Cadair) [inline docs/intro/acquiring_data/index.rst:17]: This feels out of place in this list lol

File: docs/intro/acquiring_data/index.rst [hunk 6]
```diff
@@ -4,14 +4,534 @@
...
+
+The `~sunpy.net.Fido` object is in `sunpy.net`.
+All the examples in this guide use ``Fido``, so lets start by importing it::
+
+    >>> from sunpy.net import Fido, attrs as a
+
+.. contents::
+    :depth: 3
+
+Fido supports a number of different remote data sources. To see a list the Fido object can be printed::
```
Comment (Cadair) [inline docs/intro/acquiring_data/index.rst:33]:
```suggestion
Fido supports a number of different remote data sources.
To see a list the Fido object can be printed::
```

File: docs/intro/acquiring_data/index.rst [hunk 7]
```diff
@@ -4,14 +4,534 @@
...
+Some other attributes are client specific, and are found under client specific submodules, e.g. `attrs.vso <sunpy.net.vso.attrs>` and `attrs.jsoc <sunpy.net.jsoc.attrs>`.
+The full list of attributes can be found in the `attrs submodule reference <sunpy.net.attrs>`.
+
+Some search attributes need one or more values specifying, for example ``Time`` needs at least a start and an end date to specify a time range::
+
+    >>> a.Time('2012/3/4', '2012/3/6')
+    <sunpy.net.attrs.Time(2012-03-04 00:00:00.000, 2012-03-06 00:00:00.000)>
+
+For attributes that can take a range of different values, printing the attribute lists the values sunpy knows about.
+These values are updated with every release of sunpy, so may not be quite up to date!
```
Comment (Cadair) [inline docs/intro/acquiring_data/index.rst:85]:
```suggestion
These values are updated with every release of sunpy, so may not be quite up to date.
```

File: docs/intro/acquiring_data/index.rst [hunk 8]
```diff
@@ -4,14 +4,534 @@
...
+    be_halpha                   VSO         BE-Halpha                INAF-OACT Barra Equatoriale Hα Instrument
+    bigbear                     VSO         Big Bear                 Big Bear Solar Observatory, California TON and GONG+ sites
+    caii                        VSO         CAII                     Kanzelhöhe Ca II k Instrument
+    cds                         VSO         CDS                      Coronal Diagnostic Spectrometer
+    celias                      VSO         CELIAS                   Charge, Element, and Isotope Analysis System
+    ...
+
+This is a full list of known values, a description, and which clients support those values (if you want to search using a specific data source).
+Printing attributes like this is supported for most attributes, including  client specific ones.
+These attributes also support tab-completion to auto to auto-fill the attribute name, for example typing ``a.jsoc.aia_f<TAB>`` in a jupyter notebook will show you the available attributes that start with "aia_f".
```
Comment (Cadair) [inline docs/intro/acquiring_data/index.rst:107]:
```suggestion
These attributes also support tab-completion to auto to auto-fill the attribute name, for example typing ``a.jsoc.aia_f<TAB>`` will show you the available attributes that start with "aia_f".
```

**Review by Cadair — CHANGES_REQUESTED** [review]

**Comment by nabobalis** [comment]
Looks like a legit doc test failure

```
_____________________________ [doctest] index.rst ______________________________
[gw0] linux -- Python 3.11.2 /home/runner/work/sunpy/sunpy/.tox/py311/bin/python
027     >>> from sunpy.net import Fido, attrs as a
028 
029 .. contents::
030     :depth: 3
031 
032 Fido supports a number of different remote data sources. To see a list the Fido object can be printed:
033 
034 .. code-block:: python
035 
036     >>> print(Fido)
```

**Pull request merged**